src/zenml/integrations/tekton/orchestrators/tekton_orchestrator.py

```diff
@@ -76,6 +89,16 @@ def settings_class(self) -> Optional[Type["BaseSettings"]]:
         """
         return TektonOrchestratorSettings
 
+    @property
+    def is_local(self) -> bool:
```

Comment:
The same property existed on the Kubeflow orchestrator and was removed in this PR. I'd say if there's not logic inside and it just forwards to the config property we can probably remove this? Same for the `kubernetes_context` property here and on other orchestrators?